packages/bolt-connection/src/connection-provider/connection-provider-routing.js

```diff
@@ -663,6 +682,28 @@ export default class RoutingConnectionProvider extends PooledConnectionProvider
       routingTable.forgetRouter(address)
     }
   }
+
+  _channelSsrCallback (isEnabled, action) {
+    if (action === 'OPEN') {
+      if (isEnabled === true) {
+        this._withSSR = this._withSSR + 1
+      } else if (isEnabled === false) {
+        this._withoutSSR = this._withoutSSR + 1
+      }
+    } else if (action === 'CLOSE') {
+      if (isEnabled === true) {
+        this._withSSR = this._withSSR - 1
+      } else if (isEnabled === false) {
+        this._withoutSSR = this._withoutSSR - 1
+      }
```

Comment:
```suggestion
      if (isEnabled === true) {
        this._withSSR = this._withSSR + 1
      } else {
        this._withoutSSR = this._withoutSSR + 1
      }
    } else if (action === 'CLOSE') {
      if (isEnabled === true) {
        this._withSSR = this._withSSR - 1
      } else {
        this._withoutSSR = this._withoutSSR - 1
      }
```
Nit: I suggest to fall back to assuming no SSR on the connection if the server is sending garbage.
This is not a blocker though as I don't expect this to happen (requires a buggy server).